anime_downloader/sites/yify.py

```diff
@@ -0,0 +1,45 @@
+import requests, json, os
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+from bs4 import BeautifulSoup
+from io import StringIO
+
+class Yify(Anime, sitename = 'yify'):
+    sitename = 'yify'
+    url = f'https://{sitename}.mx'
+    @classmethod
+    def search(cls, query):
+        search_results = requests.get(f'https://yts.mx/ajax/search?query={query}').text
```

Comment:
Use `helpers.get` and `helpers.soupify` instead of using `requests` and bs4 directly